script/css/build.js

```diff
@@ -1,7 +1,7 @@
 var isProduction = require('../helper').isProductionBuild();
 var execCommand = require('../helper/exec-command');
 
-var command = 'npm run stylelint && npm run scss && npm run autoprefixer';
```

Comment:
Why the removal of stylelint? I see it is still part of the installed packages.

Reply:
readded